index.sh

```diff
@@ -0,0 +1,6 @@
+function test () {
+  set -e
+  echo Bash console...
+
+  aws --version
+}
```

Comment:
Is there some code that actually runs this? I'm not an expert at bash but do you need to have a function call outside of the function definition for it to run?

Reply:
So I added 3 comments referring to 3 sequential steps that take place when running a test to ensure the bash layer is working.